scripts/add_electricity.py

```diff
@@ -338,6 +348,24 @@ def attach_wind_and_solar(
                 p_max_pu=ds["profile"].transpose("time", "bus").to_pandas(),
             )
 
+        if suptech == "offwind":
+            continue
+            # df.carrier.mask(df.technology=="Offshore", "offwind-ac",inplace=True)
```

Comment:
Why was this commented out?

Reply:
Hi, @pz-max. I have commented it out to be consistent with previous lines, where offwind is comemnted. Please, see figure below. This portion also need to be modified in the future, because offwind can be AC or DC, but I have not seen in PPM csvs the difference. Not stated if ac or dc.
![image](https://github.com/pypsa-meets-earth/pypsa-earth/assets/113768325/4259bae9-ab39-440e-bc0a-249cc46c82ac)
![image](https://github.com/pypsa-meets-earth/pypsa-earth/assets/113768325/e9f7377c-9adf-48d5-93fa-289eb9d9c4d9)

Reply:
@yerbol-akhmetov the commented parts for offwind was a hack for updating capital costs.
Here we want to add the capacities of all wind and solar related technologies. 
We need to keep this line/ not skipping the addition of powerplants for offwind.

My suggestion is to do something like:

From:
```
        if suptech == "offwind":
            continue
            # df.carrier.mask(df.technology=="Offshore", "offwind-ac",inplace=True)
```

To:
```
      if tech == "offwind-ac": 
          # add all offwind wind power plants by default as offwind-ac
          df.carrier.mask(df.technology=="Offshore", "offwind-ac",inplace=True)
```